Sources/SwifterSwift/SwiftStdlib/StringExtensions.swift

```diff
@@ -957,6 +957,17 @@ public extension String {
         return range(of: pattern, options: .regularExpression, range: nil, locale: nil) != nil
     }
     #endif
+    
+    #if canImport(Foundation)
+    /// SwifterSwift: Overload Swift's 'contains' operator for matching regex pattern
+    ///
+    /// - Parameter lhs: String to check on regex pattern.
+    /// - Parameter rhs: Regex pattern to match against.
+    /// - Returns: true if string matches the pattern.
+    static func ~= (lhs: String, rhs: String) -> Bool {
+        return lhs.range(of: rhs, options: .regularExpression, range: nil, locale: nil) != nil
```

Comment:
You can use the default values.
```suggestion
        return lhs.range(of: rhs, options: .regularExpression) != nil
```